src/com/hms_networks/americas/sc/taginfo/package.html

```diff
@@ -3,8 +3,12 @@
 Ewon Flexy tag information library created by the HMS Networks MU Americas Solution Center.
 This library is dependent on the HMS Networks MU Americas Solution Center library for strings,
 found at https://www.github.com/hms-networks/sc-string-lib.
+This library is dependent on the HMS Networks MU Americas Solution Center library for file
+utilities, found at https://github.com/hms-networks/sc-flexy-file-util-lib.
+This library is dependent on the HMS Networks MU Americas Solution Center library for JSON
+management, found at https://github.com/hms-networks/sc-flexy-json-lib.
```

Comment:
Also should be reflected in the README

Reply:
This can be added to the README as part of the pull request which will release this version.